common/lib/container-definitions/src/loader.ts

```diff
@@ -93,6 +94,28 @@ export interface IContainerEvents extends IEvent {
     (event: "dirty" | "saved", listener: (dirty: boolean) => void);
 }
 
+export namespace ConnectionState {
```

Comment:
This doesn't seem to be correctly pulling out documentation. Says it's not documented in the .api.md file

Reply:
Yeah it needed an additional comment for the namespace along with the type, should be fixed now!